ts/parts/StockChart.ts

```diff
@@ -1092,3 +1102,37 @@ addEvent(Chart, 'update', function (
         delete options.scrollbar;
     }
 });
+
+// Extend the Axis prototype to calculate start min/max values
+// (including min/maxPadding). This is related to using vertical panning
+// (#11315).
+addEvent(Axis, 'afterSetScale', function (
+    this: Highcharts.Axis
+): void {
+    var axis = this,
+        panning = (axis.chart.options.chart as any).panning,
+        options = axis.options as Highcharts.PanAxisOptions;
+
+    if (
+        panning &&
+        (panning.type === 'y' ||
+        panning.type === 'xy') &&
+        !axis.isXAxis &&
+        !defined(options.startMin) &&
+        !defined(options.startMax)
+    ) {
+
+        var min = Number.MAX_VALUE,
+            max = Number.MIN_VALUE;
+
+        axis.series.forEach(function (series): void {
+            min = Math.min(Math.min.apply(null, (series.yData as any)), min) -
```

Comment:
Maybe it's not relevant, but we have `H.arrayMin` to be [safe](https://github.com/highcharts/highcharts/blob/3f9a9b8f4d52283f41dec3585d311acd02a0bd78/ts/parts/Utilities.ts#L2127-L2150).